src/app_service/service/community/service.nim

```diff
@@ -784,6 +784,9 @@ QtObject:
     else:
       result.sort(sortDesc[ChatDto])
 
+  proc getChatsByIds*(self: Service, chatIds: seq[string]): seq[ChatDto] =
+    return self.chatService.getChatsByIds(chatIds)
```

Comment:
Instead of adding this in the community service, it probably makes more sense to pass the chat service to the community controller instead